src/main/webapp/intrigue-api/metacards/metacards.js

```diff
@@ -238,9 +238,45 @@ const createThumbnailUrl = result => {
   return 'data:image/jpeg;base64,' + thumbnail
 }
 
+const getResultFormAttributes = async (resultFormID, catalog) => {
+  if (!resultFormID || resultFormID === 'All Fields') return null
+  try {
+    const filterTree = {
+      type: 'AND',
+      filters: [
+        { type: '=', property: 'id', value: resultFormID },
+        { type: 'LIKE', property: 'metacard-tags', value: '%' },
+      ],
+    }
+    const q = { filterTree }
+    const json = await catalog.query(processQuery(q))
```

Comment:
Why not use 
```suggestion
    const res = await metacardsById(parent, {ids: [resultFormID]}, context)[0]
```

Reply:
Updated to use the `metacardsById` resolver